legion/legion/external/edi.py

```diff
@@ -276,3 +292,18 @@ def build_client(args):
 
     client = EdiClient(host, user, password, token)
     return client
+
+
+def build_client_from_env():
```

Comment:
We already have function `legion.external.edi.build_client` you should modify existed function to allow use it without arguments